contrib/core/actions/sendmail.yaml

```diff
@@ -41,5 +41,10 @@ parameters:
     type: string
   sudo:
     immutable: true
+  attachments:
+    description: Array of attachment file paths, comma-delimited.
+    position: 6
+    required: false
+    type: "string"
```

Comment:
As mentioned on Slack, I would prefer to make this an array and serialize to comma delimited string in the script runner.

In any case, I can take care of that later.